docs/iot/iot-partners/anchor.md

```diff
@@ -0,0 +1,43 @@
+# Tizen IoT with Anchor SoM Module
+
+> The Fastest Way to design your IoT Applications
```

Comment:
@mijin85cho Please suggest:

Here we suggest providing more information regarding SoM in general for IoT and how Anchor is helping in this scenario/market with Tizen’s power. Then, end the page with call-to-actions to reference links for Anchor SoMs series. 

Whereas currently, this page has copied content from the specification document of Anchor3 SoM: https://drive.google.com/open?id=13GqlZtnRxxGUqiYBCiqjyx1hd_yJ4tIr

This will become an issue of plagiarism, so please confirm regarding the copyrights for the content with Anchor SoM officials.

Kindly check the PR #839 where this kind of issue is mentioned in the comments. May be we can also do the same.

Reply:
How can i confirm it?
Now I'm in charge of Anchor Series Product in Dignsys Inc. that owns copyright.

Reply:
The committer(johnny1411) is from Dignsys.